src/CommandAndStateMessageParser.cpp

```diff
@@ -611,14 +614,25 @@ RigidBodyState CommandAndStateMessageParser::getCameraHeadTiltMotorStateRBS(
 
     RigidBodyState rbs;
     rbs.time = Time::now();
-    auto yaw = root["tilt"]["position"].asDouble() * M_PI / 180;
+    auto tilt = root["tilt"]["position"].asDouble() * M_PI / 180;
```

Comment:
camera_head2world_tilt

Reply:
Do you mean passing the JointState in the caller as an argument to the getCameraHead* methods? If so, the caller is the Task, it doesnt have access to the JSON, which is a private member. For the second option, I think that's better as well.